examples/example-gateway/middlewares/middleware-config.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
Let's make this file an object with a nested field.

The work @Matt-Esch is doing on the module system requires configs to be objects.

```
{
  middlewares: [...]
}
```